index.js

```diff
@@ -1,5 +1,18 @@
 "use strict";
 var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
 fluid.module.register("gpii-pouchdb", __dirname, require);
 
-require("./src/js/pouch");
\ No newline at end of file
+require("./src/js/pouch");
+
+fluid.registerNamespace("gpii.pouch");
+
+gpii.pouch.loadTestingSupport = function () {
+    // require("gpii-express");
```

Comment:
Previously, the errors were revealed by running `tests/js/launch-test-harness.js`, which would immediately complain about the lack of tests and exit.  This is no longer the case, and I have uncommented the lines.